src/components/Replay.js

```diff
@@ -13,6 +13,7 @@ import {
   Picker
```

Comment:
Oops, didn't mean to leave that in there. Thank you for catching it. Ill send another PR with it.